### PR TITLE
docs: complete macOS node-gyp troubleshooting steps in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by DiscRisknBisque**
Does what it says on the tin.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,14 @@ If installation fails due to a `node-gyp` issue you may need to perform some add
 
 ### on macOS
 
-- I have no idea.
+- Determine if you have Xcode command line tools installed
+  - example the console will tell you if they're already installed): `xcode-select --install`
```
Comment (davidmurdoch) [inline CONTRIBUTING.md]: Can you make the steps more prescriptive? i.e., tell the user what to do directly.

**Review by davidmurdoch — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (DiscRisknBisque) [reply CONTRIBUTING.md]: This style (minus the missing "(") matches the other instructions. Would you like me to modify those too? They're short so it wouldn't take long.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,14 +31,17 @@ If installation fails due to a `node-gyp` issue you may need to perform some add
...
-- Determine if you have Xcode command line tools installed
-  - example the console will tell you if they're already installed): `xcode-select --install`
-- Determine if you have Python 2.7 installed
-  - example: `which python2.7`
-- If you do not have Python 2.7 installed, you need to install it. We recommend `pyenv` for this so you can keep the default MacOS python installation untouched.
+- Attempt to install Xcode command line tools (the console will tell you if they're already installed)
+  - example: `xcode-select --install`
+- Determine if you have Python 2.x installed
+  - example: `which python`
+- If you do not have Python 2.x installed, you need to install it. We recommend `pyenv` for this so you can keep the default MacOS python installation untouched.
```
Comment (davidmurdoch) [inline CONTRIBUTING.md]: I don't know Python things... but does this need to be Python `2.7.x`?

Comment (davidmurdoch) [inline CONTRIBUTING.md]:
```suggestion
- If you do not have Python 2.x installed, you need to install it. We recommend `pyenv` for this so you can keep the default macOS Python installation untouched.
```

Comment (jeffsmale90) [reply CONTRIBUTING.md]: `which python` won't necessarily tell you which version is installed (for example, if you use pyenv, it'll just show you the shim).

`python --version` is probably more useful.

**Review by jeffsmale90 — CHANGES_REQUESTED** [review]

**Review by MicaiahReid — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,22 +16,39 @@
 
 If installation fails due to a `node-gyp` issue you may need to perform some additional system configuration.
 
-### on Linux (Ubuntu-based)
+Ganache uses [node-gyp v7.1.2](https://github.com/nodejs/node-gyp/tree/v7.1.2) as part of it's build system, which requires Python v2.7, v3.5, v3.6, v3.7, or v3.8 to be installed on the system.
```
Comment (davidmurdoch) [inline CONTRIBUTING.md]:
```suggestion
Ganache uses [node-gyp v7.1.2](https://github.com/nodejs/node-gyp/tree/v7.1.2) as part of its build system, which requires Python v2.7, v3.5, v3.6, v3.7, or v3.8 to be installed on the system.
```

**Review by jeffsmale90 — APPROVED** [review]

**Review by davidmurdoch — APPROVED** [review]

**Comment by davidmurdoch** [comment]
image

@MicaiahReid <3

**Pull request merged**